hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/HoodieCopyOnWriteTableInputFormat.java

```diff
@@ -261,6 +263,20 @@ private List<FileStatus> listStatusForSnapshotMode(JobConf job,
     return targetFiles;
   }
 
+  protected boolean checkIfValidFileSlice(FileSlice fileSlice) {
+    Option<HoodieBaseFile> baseFileOpt = fileSlice.getBaseFile();
```

Comment:
checkIfValidFileSlice -> filterValidFileSlice ?

Reply:
> checkIfValidFileSlice -> filterValidFileSlice ?

I'm not sure that if a boolean returned method return may throw an IllegalXXXXException, it may be more appropriate to be named with checkXXXX. Is it?

Reply:
As per the naming convention, `isValidFileSlice` sounds better. Also, I notice that the override of this method in `HoodieMergeOnReadTableInputFormat` is logically same. So, why do we need to override?

Reply:
Sorry, i misread. I take it back. It is not the same. I think this is good to land. Any method rename you can take up in a subsequent PR as we're pretty close to the release.

Reply:
> As per the naming convention, `isValidFileSlice` sounds better. Also, I notice that the override of this method in `HoodieMergeOnReadTableInputFormat` is logically same. So, why do we need to override?

Re: Also, I notice that the override of this method in `HoodieMergeOnReadTableInputFormat` is logically same. So, why do we need to override?

 If do not override `checkIfValidFileSlice()`, the valid file slice in mor which match the condition `!baseFileOpt.isPresent() && latestLogFileOpt.isPresent()` will be filtered.